docs/blueprints/002-extension-registry.md

```diff
@@ -0,0 +1,526 @@
+          service: git+https
+          description: Gitea git/https API
+          endpoints:
+            - name: public
+              url: https://mlflow-minio:9000
+              type: external
+              default: True
+      profiles:
+        - name: default
+          scope: global
+          configuration:
+            - name: username
+              value: fuseml
+            - name: password
+              value: 8KqS5xWQ4eagRu
+  ```
+
+4. example showing that the extension root element can be used to host different instances of different tools/services, even though this is not recommended:
+
+  ```yaml
+  name: s3-storage
+  product: s3
+  description: S3 storage services
+  instances:
+    - name: minio
+      version: "4.1.3"
+      location: local
+      services:
+        - name: s3
+          service: s3
+          description: Minio S3 storage deployed locally
+          profiles:
+            - local-minio
+          endpoints:
+            - name: cluster
+              url: http://mlflow-minio:9000
+              type: internal
+            - name: ingress
+              url: http://mlflow.10.110.120.130.nip.io
+              type: external
+              default: True
+    - name: aws
+      version: "4.1.3"
+      location: local
+      services:
+        - name: aws
+          service: s3
+          description: AWS S3 storage
+          profiles:
+            - AWS-S3
+          endpoints:
+            - name: s3.amazon.com
+              url: https://s3.amazonaws.com
+              type: external
+              default: True
+      profiles:
+        - name: local-minio
+          scope: global
+          configuration:
+            - name: AWS_ACCESS_KEY_ID
+              value: 24oT0SfbJPEu6kUbUKsH
+            - name: AWS_SECRET_ACCESS_KEY
+              value: cMGiZff8KqS5xWQ4eagRujh1tDcbQyRP0bEJSBOf
+        - name: aws
+          scope: global
+          configuration:
+            - name: AWS_ACCESS_KEY_ID
+              value: sWRS24oT0SfbJPEu6kU3EWf
+            - name: AWS_SECRET_ACCESS_KEY
+              value: abl4SDcMGiZff8KqS5xWQ4eagRujh1tDcbQyRP0s
+  ```
+5. KFServing as an example of a k8s controller running in the same cluster as FuseML
+
+  ```yaml
+  name: kfserving
+  product: kfserving
+  description: KFServing prediction service
+  instances:
+    - name: local
+      version: "0.6.0"
+      location: local
+      services:
+        - name: API
+          service: kfserving-api
+          description: KFServing CRDs
+          endpoints:
+            - name: cluster
+              type: cluster-api
+              default: True
+        - name: UI
+          service: kfserving-ui
+          description: KFServing UI
+          endpoints:
+            - name: ui
+              url: http://kfserving.10.120.130.140.nip.io/
+              type: external
+    ```
+
+6. Seldon core as an example of a k8s service running in another cluster as FuseML and more tightly regulated by the admin
```

Comment:
Some formatting is broken: starting at this point, it's all shown as a code...

Reply:
fixed